src/Symfony/Component/HttpKernel/Tests/HttpCache/EsiTest.php

```diff
@@ -92,6 +92,17 @@ public function testProcessDoesNothingIfContentTypeIsNotHtml()
         $this->assertFalse($response->headers->has('x-body-eval'));
     }
 
+    public function testRemoveEsiRemoveBlocks()
```

Comment:
esi:comment doesn't accept content, it only has a text attribute.

http://www.w3.org/TR/esi-lang
`<esi:comment text="Include an ad"/>`